package.json

```
@@ -79,6 +79,9 @@
     "@storybook/channels": "6.0.28",
     "@storybook/core": "6.0.28",
     "@storybook/react": "6.0.28",
+    "@testing-library/react": "10.4.9",
+    "@testing-library/react-hooks": "5.0.3",
+    "@testing-library/jest-dom": "5.11.9",
```

nit to keep them in alphabetical order:

```suggestion
    "@testing-library/jest-dom": "5.11.9",
    "@testing-library/react": "10.4.9",
    "@testing-library/react-hooks": "5.0.3",
```